### ISSUE TITLE
Ignore directive false positive

### ISSUE THREAD

**Issue by bartlomieju** [denoland/deno_lint#282]
```ts
export function deepAssign(
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  target: Record<string, any>,
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  ...sources: any[]
): // eslint-disable-next-line @typescript-eslint/ban-types
object | undefined {
```
This code still produces diagnostic:
```
(ban-types) Use `Record<string, unknown>` instead
object | undefined {
~~~~~~
    at /Users/biwanczuk/dev/deno/std/_util/deep_assign.ts:22:0
```
